### ISSUE TITLE
Allow static images

### ISSUE THREAD

**Issue by brannondorsey** [kibio/kibio#14]
PNG, JPG, TIFF, GIF (oh my!), etc...

**Comment by bakercp** [comment]
@brannondorsey Are you proposing that this be in place of a video layer?

**Comment by brannondorsey** [comment]
Yes, that you would be able to add still images instead of videos if you were inclined. This would make it a more versatile installation tool I think.
